test-clients/src/main/java/com/hedera/services/bdd/suites/perf/CryptoTransferLoadTest.java

```diff
@@ -113,6 +115,7 @@ protected HapiApiSpec runCryptoTransfers() {
 						fileUpdate(APP_PROPERTIES)
 								.payingWith(GENESIS)
 								.overridingProps(Map.of("hapi.throttling.buckets.fastOpBucket.capacity", "1300000.0")),
+						reduceFeeFor(CryptoTransfer, 2L, 3L, 3L),
```

Comment:
Can the fees be reduced to even lower, or to all `0`s? For longer run and higher TPS, I am afraid the small fees may still adds up to deplete the payer balance before the perf test ends.

Reply:
No, we wouldn't want that because we still want to see fees (all kinds including node, network and service fees) being charged. Besides, as I just added in the comment of the PR, a 3-hr test used less than 2 hbars!
`2021-03-03 21:23:07.913 INFO   188  HapiGetAccountBalance - 'RunCryptoTransfers' - balance for 'sender': 99999890913892 tinyBars (999998ħ)`

Reply:
I thought for a dedicated 3-hr transfer load test with 10K or higher TPS, the fee will be much higher. But a quick calculation shows it will be less than 9 hbar for its theoretical max value.

Reply:
Exactly, and since we are running 6N-6C, the payer account for each client will be deducted less than 2 hbars, as mentioned.